build-paths.json

```diff
@@ -6,13 +6,17 @@
         "core_deps": [
             "bower_components/es5-shim/es5-shim.js",
             "bower_components/jquery/dist/jquery.js",
-            "bower_components/jquery-ui/ui/jquery-ui.custom.js",
+            "bower_components/jquery-ui/ui/jquery.ui.core.js",
+            "bower_components/jquery-ui/ui/jquery.ui.widget.js",
+            "bower_components/jquery-ui/ui/jquery.ui.mouse.js",
+            "bower_components/jquery-ui/ui/jquery.ui.draggable.js",
```

Comment:
Please make sure ?blocks still works (structured-blocks), as that was a big user of jQuery.UI parts.